packages/allure-mocha/test/runner.ts

```diff
@@ -5,16 +5,9 @@ import path from "path";
 import glob from "glob";
 import "source-map-support/register";
 
-declare module "mocha" {
-  interface InterfaceContributions {
-    "mocha-typescript": never;
-  }
-}
-
 const mocha = new Mocha({
-  ui: "mocha-typescript",
   timeout: 16000,
-  reporter: "mocha-multi",
+  reporter: "list",
```

Comment:
Mocha-multi was used here to generate a demo report from the tests. Can we keep this?

Reply:
I did it just because it uses only root specs in report. So we don't see any specific features under test (fixtures). But no problem, will revert.